pyfr/plugins/base.py

```diff
@@ -178,6 +195,22 @@ def __init__(self, intg, *args, **kwargs):
 
 
 class SurfaceMixin:
+    def __init__(self, intg, *args, **kwargs):
```

Comment:
Is this something we always want to do?  I do not think the fluid force needs this functionality, for example, yet it is in the constructor.

Reply:
I can move it to another method, it doesn't depend on the `ele_map` so it should be fine to call anytime.